pages/edge/learn/volumes.mdx

```diff
@@ -145,3 +145,7 @@ wasmer app volumes rotate-secrets
 ```
 
 This command will rotate their values and show the new ones directly in the CLI. 
+
+### Data Restoration Policy
+
+We backup volume data every 24 hours. In the unlikely event of data loss (or if you want to restore older data), you can restore your data with a maximum recovery point objective (RPO) of 24 hours, meaning the most you would lose is one day's worth of data.
```

Comment:
Let's link the support page here if a user wants to restore a reset: https://docs.wasmer.io/edge/support

(Note: we really need to set up some support system with email, but for now a link should be fine)